pipeline-model-definition/src/test/resources/whenEnvFalse.groovy

```diff
@@ -1,7 +1,7 @@
 /*
  * The MIT License
  *
- * Copyright (c) 2016, CloudBees, Inc.
+ * Copyright (c) 2017, CloudBees, Inc.
```

Comment:
Sorry, I used to work for an enterprise that made it's living on copyrights and patents, so I've been thoroughly indoctrinated on the matter ;)